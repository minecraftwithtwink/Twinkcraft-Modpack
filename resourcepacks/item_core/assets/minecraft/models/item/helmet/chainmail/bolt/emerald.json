{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/helmet/base/chainmail/bolt",
        "layer1": "minecraft:trims/items/helmet_trim/bolt_emerald"
    }
}