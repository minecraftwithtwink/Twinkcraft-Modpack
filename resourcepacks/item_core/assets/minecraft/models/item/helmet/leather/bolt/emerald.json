{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/helmet/base/leather/bolt",
        "layer1": "minecraft:item/helmet/base/leather_overlay/bolt",
        "layer2": "minecraft:trims/items/helmet_trim/bolt_emerald"
    }
}