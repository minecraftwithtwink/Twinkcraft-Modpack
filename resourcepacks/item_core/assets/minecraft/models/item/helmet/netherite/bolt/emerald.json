{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/helmet/base/netherite/bolt",
        "layer1": "minecraft:trims/items/helmet_trim/bolt_emerald"
    }
}