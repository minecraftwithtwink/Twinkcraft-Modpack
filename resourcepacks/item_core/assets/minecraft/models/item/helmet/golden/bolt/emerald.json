{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/helmet/base/golden/bolt",
        "layer1": "minecraft:trims/items/helmet_trim/bolt_emerald"
    }
}